contrib/babelfishpg_tsql/src/procedures.c

```diff
@@ -2442,7 +2442,7 @@ sp_addrolemember(PG_FUNCTION_ARGS)
 					 errmsg("User or role '%s' does not exist in this database.", membername)));
 
 		/* Map the logical role name to its physical name in the database. */
-		physical_role_name = get_physical_user_name(get_cur_db_name(), lowercase_rolname);
+		physical_role_name = get_physical_user_name(get_cur_db_name(), lowercase_rolname, true);
```

Comment:
pfree before re-assignment

Reply:
We did not do this in 4X. Should we still do it here ?

Reply:
Actually this is first user of this variable.